suitcase/utils/tests/tests.py

```diff
@@ -88,5 +88,12 @@ def test_fixture(example_data):
 
 
 def test_fixture_with_ignore(example_data):
-    "Exercise the example_data 'ignore' argument that skip some parameters."
+    "Exercise the example_data 'ignore' argument that skips some parameters."
     assert type(example_data(ignore=['bulk_events'])) is list
+
+
+def test_fixture_with_md(example_data):
+    "Exercise the example_data 'md' argument that is passed to the RunEngine."
+    documents = example_data(md={'user': 'Dan'})
+    assert type(documents) is list
+    assert dict(documents)['start']['md']['user'] == 'Dan'
```

Comment:
This doesn't seem quite right. Calling `dict()` on a list doesn't work. I think you want to pick off the first element of the list (which should be the Run Start document).

Reply:
`documents` is a list of 2-ples like `[('start', {...}), ..., ('stop', {...})]` so `dict(documents)` returns a dictionary keyed with document names. I guess I created the dictionary because at first I thought I would look at more than just the start document.

Replacing that with your suggestion.